pkg/k8s/apis/cilium.io/v1/types.go

```diff
@@ -74,6 +74,12 @@ type CiliumNetworkPolicyNodeStatus struct {
 
 	// LastUpdated contains the last time this status was updated
 	LastUpdated Timestamp `json:"lastUpdated,omitempty"`
+
+	// Revision contains the current policy revision of the node.
+	Revision uint64 `json:"localPolicyRevision,omitempty"`
+
+	// Enforcing is set if all endpoints are at the given Revision number.
```

Comment:
Enforcing is set to true once all endpoints present at the time the policy has been imported are enforcing this policy

Reply:
Done